### ISSUE TITLE
[Bug] Template tags are not rendering in layout file

### ISSUE THREAD

**Issue by manuganji** [Thomascogez/nodemailer-mjml#144]
**Describe the bug**
I use a layout with partials. I think the tags inside the layout file are not rendered properly.

**To Reproduce**

Pass in a layout with some `{{tag}}` beside the `slots`s.
```
       templateLayoutName: 'base-layout',
	   templateLayoutSlots: {
	   content: 'partials/sign-up',
	   reason: 'partials/reason'
        },
        templateData: {
	   BRAND_NAME,
	   blockUrl,
	   url,
	firstName,
	reason: `This mail is sent to you because you have signed up to ${BRAND_NAME}. If you didn't sign up, please ignore.`
```
Example layout file `base-layout.mjml`

```
<mjml>
	<mj-body>
		<mj-wrapper background-color="#efefef">
			<mj-section padding-bottom="0px" padding-top="0" text-align="left">
				<mj-column width="20%" vertical-align="middle">
					<mj-image
						src="https://{{BRAND_NAME}}/logo.svg"
						alt="{{BRAND_NAME}} Logo"
						align="left"
						border="none"
					></mj-image>
				</mj-column>
				<mj-column vertical-align="middle">
					<mj-text font-size="20px" font-weight="bold" align="left">
						{{BRAND_NAME}}
					</mj-text>
				</mj-column>
			</mj-section>
			{{ slots:content }}
		</mj-wrapper>
		{{ slots:reason }}
     </mj-body>
</mjml>

```
**Expected behavior**
Expected `BRAND_NAME` to be correctly replaced in the rendered email.

**Desktop (please complete the following information):**
 - OS: [e.g. iOS] macOS Ventura 13
 - Node version [e.g. 19.x.x] v18.11

**Additional context**
I'm using this in Svelte Kit. The same tags are correctly getting rendered inside the partials.

**Comment by Thomascogez** [comment]
Hey thanks for the repport ! Good catch !

I successfully reproduce the bug and release a fix in version latest release (v1.2.32)

**Comment by manuganji** [comment]
Thanks a lot! :) I created this issue just before going to bed. Didn't expect such a quick response. Grateful!
